docs/fundamentals/executor/instrumenting-executor.md

```diff
@@ -14,6 +14,10 @@ This section describes **custom** tracing spans. To use the Executor's default t
 The OpenTelemetry collector backend setup is described in the {ref}`OpenTelemetry Setup <opentelemetry>` section.
 ```
 
+```{hint}
```

Comment:
will be deprecated soon

Reply:
The other question is to keep or remove the `Monitoring` pages from the index? I have removed them in the current commit.

Reply:
I would keep it like it currently is (removed from the index). Otherwise it looks duplicate and I can see confusion arising. The people that need it should still be able to find it.